README.md

```diff
@@ -201,3 +201,6 @@ args = {
 ```
 `Exceptions:`
 No Exceptions (API takes care of other exceptions)
+
+#### Important Note
```

Comment:
I would rather emphasize that you need a `startPosition` if `scanDirection` is `FORWARD`.